.circleci/config.yml

```diff
@@ -50,12 +56,6 @@ jobs:
           command: ./test-ets.sh
           no_output_timeout: 1h
 
-      - run:
-          name: additional compilation & dist
-          # this step builds parts of the codebase which are not tested in CI
-          # so as to prevent compilation regression
-          command: $SBT benchmark:compile snappy:compile dist
```

Comment:
Maybe we could keep `dist` step?

Reply:
https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84 i think is already on buildkite side

Reply:
i think it was already included in buildkite: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84

Reply:
Is already included for buildkite pipeline: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84
or you mean having it as a separate step?